pulp_rpm/app/tasks/synchronizing.py

```diff
@@ -319,422 +320,469 @@ async def run(self):
         remote_url = remote_url if remote_url[-1] == "/" else f"{remote_url}/"
```

Comment:
you cannot leave this line here because it adds '/' to the url ( whether a url or mirrorlist) so error you are raising in the end will incorrectly contain slash no matter what `raise ValueError(_(f"Cannot sync content from the remote {remote_url}"))`
I suggesting moving the line into the `_fetch_remote_urls`